.github/pull_request_template.md

```diff
@@ -1,15 +1,28 @@
 # Description
 
 Please include a summary of the changes and the related issue, if it exists. 
-Also, include relevant motivation and context for this PR. List any
-dependencies that are required for this change.
+Also, include relevant motivation and context for this PR. List any dependencies that are required for this change.
 
-Fixes # (issue)
+- Fixes # (issue)
 
 ## Did you add the right label?
 
 Remember to add the right labels to this PR.
 
 ## How was this tested?
 
-Describe what kind of tests for this functionality have been added.
+Describe the types of tests that have been added for this functionality
+
+## Did you update the documentation?
+
+Remember to ask yourself if your PR introduced changes to the following documentation:
+
+- [Manual installation guide](https://github.com/trento-project/docs/blob/main/guides/manual-installation.md)?
```

Comment:
Having this list of docs is a dual edged sword, it is nice to have a list where on every pr we can shortly check out if some changes are required, but on the other hand this list needs to be maintained from time to time.

Open for suggestions if we should keep this list or only the text:
Add a documentation PR or write that no changes are required for the documentation.

Reply:
I think having the links is fine, in the end, as we will use it frequently it should be obvious fast enough when it gets outdated.

Reply:
I agree with @rtorrero. I am likely to forget checking some documents, if I do not have a list available